.github/workflows/elixir.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: mix do deps.get, test
 
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
`ubuntu-latest` now points to 22.04 for which hex.pm does not have older OTP builds.

See:

* https://repo.hex.pm/builds/otp/ubuntu-22.04/builds.txt
* https://repo.hex.pm/builds/otp/ubuntu-20.04/builds.txt